lib/atom-runner.coffee

```diff
@@ -70,11 +71,13 @@ class AtomRunner
     atom.commands.add 'atom-workspace', 'run:selection', => @run(true)
     atom.commands.add 'atom-workspace', 'run:stop', => @stop()
     atom.commands.add 'atom-workspace', 'run:close', => @stopAndClose()
+    atom.commands.add 'atom-workspace', 'run:prev', => @run(false, true)
```

Comment:
What if the previous run was a `run:selection`? The implementation should be robust enough to support any previous execution.